sdk-python/omnilogger/data/pricing.json

```diff
@@ -0,0 +1,44 @@
+{
+  "updated_at": "2023-09-20T12:00:00.000Z",
```

Comment:
How does this work if pricing changes? Do all historical data use the updated pricing? Perhaps that isn't the desired solution?